HW4_Trofimov/das_protein_tools.py

```diff
@@ -0,0 +1,361 @@
+# importing necessary modules
+import protein_dict as pd
+from random import choice
+
+
+AMINO_LETTERS = [
+    "A",
+    "C",
+    "D",
+    "E",
+    "F",
+    "G",
+    "H",
+    "I",
+    "K",
+    "L",
+    "M",
+    "N",
+    "P",
+    "Q",
+    "R",
+    "S",
+    "T",
+    "V",
+    "W",
+    "Y",
+]
+
+
+# Function to determine is the sequence is a protein or not
+def is_protein(seq: str) -> bool:
+    """
+    This function checks if the sequence is a protein or not
+
+    Arguments:
+        seq (str): A sequence of aminoacids
+
+    Output:
+        returns True or False
+    """
+    unique_chars = set(seq)
+    aminoacids = set(pd.aa_monoistopic_mass_dict.keys())
+    return bool(unique_chars <= aminoacids)
```

Comment:
Методы сравнения множеств предпочтительнее
```suggestion
    return unique_chars.issubset(aminoacids)
```

Reply:
Привет! Большое спасибо за комменты! А почему такой вариант предпочтительнее? Он же в любом случае выдает bool

Reply:
Да, в любом случае будет bool (если что, за это баллы не снижались, разумеется)
В целом обе операции одинаково эффективны и по сути, и по времени, и в реальности можно использовать любую. Я предлагаю при возможности использовать методы множеств во-первых, для того, чтобы вы научились их использованию, во-вторых, потому что если у вас огромный код, и вы не всегда помните, какого типа данных ваши переменные, то методы сетов сразу делают это ясным при чтении кода (или например, если надо что-то дописать\переписать). 
Оператор сравнения < и >, кстати, не на всех типах данных работают одинаково, как вы знаете, например, со строками там применяется другая логика